applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -10,19 +10,20 @@ menu "Bluetooth"
 
 config BLE_ACL_CONN_INTERVAL
 	int "Bluetooth LE ACL Connection Interval (x*1.25ms)"
-	default 80
+	default 32
 
 config BLE_ACL_SLAVE_LATENCY
 	int "Bluetooth LE Slave Latency"
 	default 0
 
 config BLE_ACL_SUP_TIMEOUT
 	int "Bluetooth LE Supervision Timeout (x*10ms)"
-	default 400
+	default 100
 
 config BLE_ACL_PER_ADV_INT_MIN
 	hex "Minimum periodic advertising interval"
 	range 0x0018 0x03C0
+	default 0x16 if TRANSPORT_BIS
```

Comment:
Or we just set default to 0x16 since CIS doesn't use periodic advertising?

Reply:
Done

Reply:
Done